### PR TITLE
update depend a bot

### PR DESCRIPTION
**Pull request by astridx**
I updated the node dependencies which depend a bot suggested and run 
npm ci
and 
npm test-jest
and npm test-unit

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
     "benchmark": "^2.1.4",
     "browserify": "^17.0.0",
     "canvas": "^2.6.1",
-    "chalk": "^4.1.2",
+    "chalk": "^5.0.0",
```
[the remark this answers is not in the document]
Comment (HarelM) [reply package.json:95]: I agree

[the remark this answers is not in the document]
Comment (astridx) [reply package.json]: I have reverted the change for chalk.
As an aside, we only use chalk in two files during integration tests and it's only about the colour of the console output.  What is your opinion on this: Do we need this at all?

[the remark this answers is not in the document]
Comment (HarelM) [reply package.json]: Seems redundant to me. The less dependencies we have the better we are.

**Comment by astridx** [comment]
I thought it would be less complicated to integrate libraries this way. But at the moment it only causes an additional PR and confusion. That's why I'm closing it.

**Pull request closed without merging**